lib/src/facts/osx/virtualization_resolver.cc

```diff
@@ -42,5 +42,4 @@ namespace facter { namespace facts { namespace osx {
         });
         return value;
     }
-
```

Comment:
Removing the line seems unnecessary, and doesn't match a style I see elsewhere. I'd rather just not change it.